ingest/src/main/java/com/microsoft/azure/kusto/ingest/QueuedIngestClientImpl.java

```diff
@@ -234,19 +215,15 @@ protected IngestionResult ingestFromStreamImpl(StreamSourceInfo streamSourceInfo
                     ingestionProperties.getTableName(),
                     dataFormat.getKustoValue(), // Used to use an empty string if the DataFormat was empty. Now it can't be empty, with a default of CSV.
                     shouldCompress ? CompressionType.gz : streamSourceInfo.getCompressionType());
-            ContainerWithSas container = resourceManager.getTempStorage();
-            BlobContainerClient blobContainerClient = container.getContainer();
-            // trace uploadStreamToBlob
-            Map<String, String> attributes = new HashMap<>();
-            attributes.put("container", blobContainerClient.getBlobContainerName());
-            MonitoredActivity.invoke((SupplierTwoExceptions<Void, IOException, URISyntaxException>) () -> {
-                azureStorageClient.uploadStreamToBlob(streamSourceInfo.getStream(), blobName,
-                        blobContainerClient, shouldCompress);
-                return null;
-            }, getClientType().concat(".uploadStreamToBlob"), attributes);
-            String blobPath = blobContainerClient.getBlobContainerUrl() + "/" + blobName + container.getSas();
```

Comment:
Note you seem to loose some context in this implementation (may need to be passed down to the API)

Reply:
Added context